src/Microsoft.Data.SqlClient/tests/tools/Microsoft.Data.SqlClient.TestUtilities/Config.cs

```diff
@@ -22,8 +22,6 @@ public class Config
         public string AADServicePrincipalSecret = null;
         public string AzureKeyVaultURL = null;
         public string AzureKeyVaultTenantId = null;
-        public string AzureKeyVaultClientId = null;
-        public string AzureKeyVaultClientSecret = null;
```

Comment:
You could leave these in and put an `[Obsolete]` tag on them. That way pipelines won't fail until we update them and then we can remove these properties.